src/roswire/common/msg.py

```diff
@@ -82,9 +109,51 @@ class Field:
         The name of this field.
     """
 
+    R_TYPE = r"[a-zA-Z0-9_/]+(?:\[(?:<=)?\d*\])?"
+    R_NAME = r"[a-zA-Z0-9_/]+"
+    R_VAL = r".+"
```

Comment:
Is `R_VAL` used here?

Reply:
Huh. Good point. I thought it was. It will be in ROS 2 but that's a different class.